index.js

```diff
@@ -30,6 +30,50 @@
 const loaderUtils = require('loader-utils');
 const sharp = require('sharp');
 
+const numericOptions = [
```

Comment:
The problem with enumerating all the options is that it is not very future-proof - _sharp_ may put out a new release with a new numeric option, and this will break the loader. I have proposed an alternative approach below.